src/Spago/Command/Init.purs

```diff
@@ -239,6 +240,18 @@ defaultConfig' opts =
   where
   mkDep p = Tuple (unsafeFromRight $ PackageName.parse p) Nothing
 
+withDependencies :: Config -> Dependencies -> Dependencies -> Config
+withDependencies config core test =
+  ( config
+      { package = config.package # map
+          ( \package' -> package'
+              { dependencies = core
+              , test = package'.test # map ((_ { dependencies = test }))
+              }
+          )
+      }
+  )
+
```

Comment:
This is only used for testing, could we move in that same module where it is used?